Utils/azure-toolkit-ide-libs/azure-toolkit-ide-common-lib/src/main/java/com/microsoft/azure/toolkit/ide/common/icon/AzureIcons.java

```diff
@@ -48,6 +48,7 @@ public static final class Action {
         public static final AzureIcon BROWSER = AzureIcon.builder().iconPath("/icons/action/browser").build();
         public static final AzureIcon PROPERTIES = AzureIcon.builder().iconPath("/icons/action/properties").build();
         public static final AzureIcon LOG = AzureIcon.builder().iconPath("/icons/action/log").build();
+        public static final AzureIcon TERMINAL = AzureIcon.builder().iconPath("/icons/Microsoft.Compute/actions/OpenTerminal_13x13.svg").build();
```

Comment:
there is no need to copy the icon to our repository, we can just use IntelliJ's icon directly in our code.